crochet/tests/test_setup.py

```diff
@@ -59,6 +59,19 @@ def start(self):
         self.started = True
 
 
+class FakeFunctionRegistry(object):
```

Comment:
No need for a fake, you can use the real thing... and based on comment above you don't even need to pass one in, you can just use the one inside the `EventLoop` instance.